posts/tree-cell-typing/index.qmd

```diff
@@ -0,0 +1,241 @@
+
+load("mini_nsclc")
+
+# Run InSituTree with basic options
+res <- runInSituTree(
+  x = mini_nsclc$counts,
+  neg = rowMeans(mini_nsclc$neg),
+  full_profiles = ioprofiles,
+  cth = celltype_hierarchy_list,
+  cohort = NULL,
+  excluded_genes = NULL,
+  quantile_absolute_expression_difference_param = 0.5, # Include genes in the top 0.5 fraction in absolute expression differences
+  quantile_percent_expression_difference_param = 0.5, # Include genes in the top 0.5 fraction in percentage expression differences
+  return_summary_annotation = TRUE
+)
+
+# Access the final summarized cell typing results
+head(res$summaryAnnotation, 10)
+
+```
+
+```{r echo = FALSE, warning = FALSE, eval = TRUE}
+
+res <- readRDS("./assets/small_results.rds")
+
+head(res, 10)
+```
+
+# Additional recommendations {#sec-additional-recommendations}
+The structure of the cell type hierarchy list is a key determinant of how well this approach works. Major cell types that are unnecessarily broad may lead to poor cell typing if the average gene expression is too distinct from the expression of individual cells. As an example, the cell type lists available at our [CosMx Cell Profile library](https://github.com/Nanostring-Biostats/CosMx-Cell-Profiles/tree/main){target="_blank"} often contain an initial grouping of cell types into 'Tissue' or 'Immune'; we would recommend removing that large group and providing the next level down as top annotations. `InSituTree` does run quickly, so in cases of uncertainty we would recommend running the algorithm with multiple cell type hierarchies and choosing the one that gives the best cell typing. Evaluate the results as cell typing is typically evaluated: spatial distribution of cell types, top markers for each cell type, and clustering of similar cell types in UMAP space.
+
+The approach above only works for fully supervised cell typing. In datasets where you have novel cell types not captured in your reference profile, one option is first to run semi-supervised `InSituType` on your whole dataset. Second, you could either create a reference profile from your CosMx data and run `InSituTree` using these new cell type definitions. Alternatively, you could select out only cells of interest from your full dataset cell typing annotations, for example only the immune cells, and run targeted subclustering with `InSituTree` on those cells only.
+
+
+
+# Conclusions {#sec-conclusions}
+The nested approach to supervised cell typing presented here adds another option for cell typing, particularly useful when looking to identify fine cell subtypes. This approach is one of many, and users are encouraged to consider this tool in concert with other approaches including unsupervised clustering, [gene smoothing](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/posts/marker-gene-smoothing/), [cell type refinement](https://github.com/Nanostring-Biostats/InSituType/blob/main/FAQs.md), and label transfer approaches.
```

Comment:
adds **another layer of control** on cell typing process

Reply:
Modified